apis/python/src/tiledbsoma/soma_collection.py

```diff
@@ -398,3 +380,7 @@ def _show_metadata(self, recursively: bool = True, indent: str = "") -> None:
                         raise Exception(
                             f"Unexpected object_type found: {object_type} at {obj.uri}"
                         )
+
+
```

Comment:
please add a little comment here about this class' value-add

Reply:
done